src/github/webhook.py

```diff
@@ -192,4 +193,8 @@ def handle_github_webhook(event_type, payload) -> HttpResponse:
     # if Github's data consistency needs a bit of time (does not have
     # read-after-write consistency)
     time.sleep(2)
-    return _events_map[event_type](payload)
+    try:
+        return _events_map[event_type](payload)
+    except DynamoDBLockError as e:
```

Comment:
alternatively, we could throw these errors and handle them as part of the queue retry logic. But I believe that's currently turned off?

Reply:
queue logic is actually back on! so we could throw these